docs/rules/prefer-const.md

```diff
@@ -0,0 +1,50 @@
+# Suggest to use `const` (prefer-const)
+
+If a variable is never changed, using the `const` declaration is better.
+
+## Rule Details
+
```

Comment:
Can you add a short explanation about what this rule is doing here? Something like "This rule is aimed to flag variables that are declared using `let` keyword, but never modified after declaration"